### PR TITLE
self.metadata field was overwritten to False

### PR DESCRIPTION
**Pull request by tarikaltuncu**
It was overwritten to false and this makes contradiction to documents

### CODE REVIEW

File: gensim/corpora/wikicorpus.py [hunk 1]
```diff
@@ -512,7 +512,7 @@ class WikiCorpus(TextCorpus):
 
     def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), dictionary=None,
                  filter_namespaces=('0',), tokenizer_func=tokenize, article_min_tokens=ARTICLE_MIN_WORDS,
-                 token_min_len=TOKEN_MIN_LEN, token_max_len=TOKEN_MAX_LEN, lower=True):
+                 token_min_len=TOKEN_MIN_LEN, token_max_len=TOKEN_MAX_LEN, lower=True, metadata=True):
```
Comment (piskvorky) [inline gensim/corpora/wikicorpus.py]: This seems to change the existing behaviour from `False` to `True`. Why this change?

**Comment by menshikh-iv** [comment]
hi @tarikaltuncu

> this makes contradiction to documents

what's document?

also, please answer to https://github.com/RaRe-Technologies/gensim/pull/2076#discussion_r204221096 comment

**Comment by tarikaltuncu** [comment]
Hi, on double check either I was confused or the documentation has been fixed. So it seems as it should be now.
